app/assets/stylesheets/application.css.scss

```diff
@@ -3,6 +3,10 @@
 @import 'required';
 @import 'vendor';
 @import 'vendor-overrides';
+// Remove after `basscss-sass/base-forms` is no longer in use (LG-3876). BassCSS applies default
+// styling to label elements, that this exists to unset to avoid conflicts with design system.
```

Comment:
Same point as in https://github.com/18F/identity-idp/pull/5944#discussion_r806148421 about comment not really applying.

```suggestion
```